velox/exec/Spill.h

```diff
@@ -321,6 +321,8 @@ class SpillState {
   /// results.
   SpillState(
       common::GetSpillDirectoryPathCB getSpillDirectoryPath,
+      const common::UpdateSpilledBytesAndCheckLimitCB&
```

Comment:
Can you also help update GetSpillDirectoryPathCB to pass by const reference if the caller doesn't call move to pass? Thanks!